keystone_api/apps/users/managers.py

```diff
@@ -21,7 +21,13 @@ class UserManager(BaseUserManager):
     """Object manager for the `User` database model"""
 
     def create_user(
-        self, username: str, first_name: str, last_name: str, email: str, password: str, **extra_fields
+            self,
+            username: str,
+            first_name: Optional[str],
```

Comment:
`Optional` is only needed for older Python versions. Modern versions use `str | None`.